tests/bluetooth/iso/prj.conf

```diff
@@ -0,0 +1,45 @@
+CONFIG_BT=y
+CONFIG_BT_ISO_BROADCASTER=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_DEVICE_NAME="ISO broadcast+ACL central"
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_ISO_SYNC_RECEIVER=y
+
+CONFIG_BT_ISO_MAX_CHAN=2
+CONFIG_BT_ISO_TX_BUF_COUNT=2
+
+# Just needs to send a uint32_t value
+CONFIG_BT_ISO_TX_MTU=23
+
+CONFIG_BT_CTLR_ISO_TX_BUFFERS=2
+
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV_SYNC=y
+CONFIG_BT_OBSERVER=y
+
+CONFIG_NCS_INCLUDE_RPMSG_CHILD_IMAGE=n
+CONFIG_BT_HCI_VS_EXT=n
+CONFIG_MAIN_STACK_SIZE=4096
+
+CONFIG_SHELL=y
+CONFIG_KERNEL_SHELL=y
+CONFIG_USE_SEGGER_RTT=y
+
+CONFIG_SHELL_RTT_INIT_LOG_LEVEL_NONE=y
+CONFIG_SHELL_BACKEND_RTT=y
+CONFIG_SHELL_BACKEND_SERIAL=n
+CONFIG_SHELL_VT100_COLORS=y
+CONFIG_SHELL_STACK_SIZE=2048
+CONFIG_SHELL_CMD_BUFF_SIZE=128
+CONFIG_SHELL_TAB_AUTOCOMPLETION=y
+
+# Getopt
+CONFIG_MINIMAL_LIBC=n
+CONFIG_NEWLIB_LIBC=y
+CONFIG_GETOPT_LIB=y
+# Need to disable devmem shell since it enforces Zephyr's internal version of getopt
```

Comment:
And the Zephyr `getopt` won't be sufficient for these tests?

Reply:
Correct. There is a missing function here.